pkg/cmd/get/clusters/clusters.go

```diff
@@ -0,0 +1,43 @@
+package clusters
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"sigs.k8s.io/kind/pkg/cluster"
+)
+
+var ClustersCmd = &cobra.Command{
+	Use:   "clusters",
```

Comment:
not to be too picky here but looks like the other commands are verbs and this one is a noun. shall we keep it as `list` like it used to be? or `get` maybe?